about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About page</title>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link
+    href="https://www.dafontfree.net/embed/Y29jb2dvb3NlLXByby10aGluLXJlZ3VsYXImZGF0YS84NS9jLzE3NjcxMS9Db2NvZ29vc2UgUHJvIFRoaW4tdHJpYWwudHRm"
+    rel="stylesheet" type="text/css" />
+  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+  <header class="header">
+
+    <nav>
+
+      <div class="topNav">
+
+        <ul class="navSocials">
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/facebook-new.png" alt="facebook" /></a>
+          </li>
+          <li><a href="#"><img src="https://img.icons8.com/fluency/48/000000/instagram-new.png" alt="instagram" /></a>
+          </li>
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/twitter--v1.png" alt="twitter" /></a></li>
+          <li><a href="#"><img src="https://img.icons8.com/glyph-neue/64/000000/github.png" alt="github" /></a></li>
+          <li><a href="#"><img src="https://img.icons8.com/color/48/000000/linkedin.png" alt="linkedin" /></a></li>
+        </ul>
+
+      </div>
+
+      <div class="lowerNav">
+
+        <div class="lowerNavContainer">
+
+          <div class="logo"> <a href="homepage.html"><img src="images/logo.jpg" alt="logo"></a>
+            <p>Nobilis</p>
+          </div>
+
+          <div class="openMenu"><img src="https://img.icons8.com/ios-filled/50/000000/menu--v1.png" alt="open menu" />
+          </div>
+
+          <ul class="mainMenu">
+            <li><a href="about.html">About Us</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Lecturers</a></li>
+            <li class="closeMenu"><img src="https://img.icons8.com/ios-glyphs/30/000000/delete-sign.png"
+                alt="close menu" /></li>
+          </ul>
```

Comment:
- [ ] Please make sure It's possible to navigate between the pages, there's a link from `Home` page to `About` page and the opposite too.

- Your Design



![b6](https://user-images.githubusercontent.com/63972670/163488535-d6d39e14-02b7-435c-ad3c-f055f82c5b69.PNG)